src/common/utils.ts

```diff
@@ -122,7 +123,19 @@ export class ByteArrayMap<T> {
 }
 
 function filePathRelativeTo(base: string, filePath: string): string {
-  return new URL(filePath, new URL(base, 'file://')).pathname;
+  // Normalize the base path to convert any Windows backslashes to forward slashes
+  // This is necessary because the URL object expects forward slashes
+  const normalizedBase = base.replace(/\\/g, '/');
+
+  // Create a URL object with the file protocol and the normalized base path
+  const baseURL = new URL(normalizedBase, 'file:///');
+
+  // Resolve the file path against the base URL
+  const fullURL = new URL(filePath, baseURL);
+
+  // Convert the URL back to a local file path
+  // On Windows, this will correctly handle the drive letter and convert to backslashes
```

Comment:
You can probably skip this and just use the output from the URL object? forward slashes do work on windows, then we don't have the browser incompatibility problem Jason mentioned